### PR TITLE
[Metricbeat/kibana/status] Add support for v8format

### PR DESCRIPTION
**Pull request by afharo**
## Proposed commit message

Added the v8-format-compliant `status.overall.level` field in the metrics.
The [v8 format was the default](https://github.com/elastic/kibana/issues/94419) starting... ehem... v8.0.0 😅 

While I was at it, I also added `status.overall.summary` for a human-friendlier explanation of the status.

## Checklist

- [x] My code follows the style guidelines of this project
- ~[ ] I have commented my code, particularly in hard-to-understand areas~
- ~[ ] I have made corresponding changes to the documentation~
- ~[ ] I have made corresponding change to the default configuration files~
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Disruptive User Impact

The new field (and values) needs to be used in alerts/visualizations. Refer to https://github.com/elastic/kibana/pull/76054 for more information about the breaking changes.

## Related issues

- Relates https://github.com/elastic/kibana/issues/94419

## Use cases

Stack Monitoring.

cc @elastic/kibana-core @elastic/stack-monitoring

### CODE REVIEW

File: metricbeat/module/kibana/status/_meta/fields.yml [hunk 1]
```diff
@@ -23,7 +23,31 @@
     - name: status.overall.state
       type: keyword
       description: >
-        Kibana overall state.
+        Kibana overall state (v7 format).
```
Comment (jennypavlova) [inline metricbeat/module/kibana/status/_meta/fields.yml:26]: The fields are not updated in the docs: [fields.asciidoc
](https://github.com/elastic/beats/blob/main/metricbeat/docs/fields.asciidoc#status-1) - I tried running `make update` and it adds them there.

**Review by jennypavlova — COMMENTED** [review]
Thanks for adding that! 
The `status_test.go` is failing locally only for `8.16.0`. Do you know if I need to set up something first before running the test? Can you please help with the manual testing steps?

**Comment by afharo** [comment]
@jennypavlova, feedback addressed in [`38fa1ae` (#40275)](https://github.com/elastic/beats/pull/40275/commits/38fa1aee172fff0a51110c7f770a6eb39776ccc3) and [`a0cfae8` (#40275)](https://github.com/elastic/beats/pull/40275/commits/a0cfae8bc1a7b3478de15a8c116788d9821da892)

**Review by jennypavlova — APPROVED** [review]
Thank you for the fixes! The test is passing now. LGTM (code review only)

**Pull request merged**